### ISSUE TITLE
Issue 2: Add index.html and script.js

### ISSUE THREAD

**Issue by rag-hav** [opencodeiiita/Contest-Countdown#2]
Add an index.html file that would be shown on the newtab page (refer to issue #1). 

The webpage should have a working timer shown prominently. Their should be a function in script.js to easily set this timer to any date and time.

**Comment by rag-hav** [comment]
contributor can also a stylesheet.css file if they wish

**Comment by criticic** [comment]
#1 & #2 have to be done together?

**Comment by rag-hav** [comment]
> #1 & #2 have to be done together?

@criticic yes both need to completed to see each other's effect. But different contributes can complete them independently.
